docs/pages/testing/structural-testing/index.md

```diff
@@ -34,31 +34,28 @@ Then, install StoryShots into your app with:
 npm i -D @storybook/addon-storyshots
 ```
 
-Then, add the following NPM script into your package.json:
-
-```json
-{
-  "scripts": {
-    "test-storybook": "storyshots"
-  }
-}
+Then, assuming you are using Jest for testing, you can create a test file `storyshots.test.js` that contains the following:
+
+```js
+import initStoryshots from '@storybook/addon-storyshots';
+
+initStoryshots({ /* configuration options */ });
 ```
 
-Now you can run the above command with:
+Now you can snapshot test all of your stories with:
 
 ```sh
-npm run test-storybook
+npm test
```

Comment:
@ndelangen i'd bet it started out this way and then the tool changed and the docs got stale